src/lib/components/marketing/MarketingNav.svelte

```
@@ -83,15 +89,15 @@
-				<a href={resolve('/#features')} class="nav-link text-[13px] font-medium no-underline"
-					>Features</a
-				>
-				<a href={resolve('/#how-it-works')} class="nav-link text-[13px] font-medium no-underline"
-					>How it works</a
-				>
-				<a href={resolve('/#pricing')} class="nav-link text-[13px] font-medium no-underline"
-					>Pricing</a
-				>
+				{#each navLinks as link (link.href)}
+					<a
+						href={link.href}
```

ESLint

[error] Unexpected href link without resolve().

(svelte/no-navigation-without-resolve)